client/interfaces.ts

```diff
@@ -0,0 +1,10 @@
+import {ISuperdeskGlobalConfig} from 'superdesk-api';
+
+export interface IAnalyticsConfig extends ISuperdeskGlobalConfig{
+    highcharts_license_type?: string;
```

Comment:
I'd be nice to have some organization instead of keeping everything in the top-level

```
export interface IAnalyticsConfig extends ISuperdeskGlobalConfig {
    highcharts: {
        license: {
            licensee?: string;
            id?: string;
            expiry?: string;
            customer_id?: string;
            contact?: string;
        }
        other_setting: string;
    };
}
```